diff --git a/dist/module.d.mts b/dist/module.d.mts
index d9a4244ce1c33216d71a5d77a3db4b06b3e3f722..f110aa685430170a3f8dc41eaffe7367535303b2 100644
--- a/dist/module.d.mts
+++ b/dist/module.d.mts
@@ -2,6 +2,7 @@ import * as _nuxt_schema from '@nuxt/schema';
 
 interface ModuleOptions {
     url: string;
+    manualInit?: boolean;
 }
 declare const _default: _nuxt_schema.NuxtModule<ModuleOptions, ModuleOptions, false>;
 
diff --git a/dist/module.mjs b/dist/module.mjs
index ff9e2c9cc18aa74ccd384f7ae64ba5a7c07b311c..f9b7eb580e989b2d714f751601cbd3d765b9cc53 100644
--- a/dist/module.mjs
+++ b/dist/module.mjs
@@ -9,7 +9,6 @@ const module = defineNuxtModule({
   defaults: {},
   setup(options) {
     const resolver = createResolver(import.meta.url);
-    addPlugin(resolver.resolve("./runtime/plugin"));
     addImports([
       { name: "useConvexClient", from: "convex-vue" },
       { name: "useConvexQuery", from: "convex-vue" },
@@ -17,12 +16,10 @@ const module = defineNuxtModule({
     ]);
     updateRuntimeConfig({
       public: {
-        convex: {
-          url: options.url
-          // auth: options.getAuth,
-        }
+        'convex-vue': options
       }
     });
+    addPlugin(resolver.resolve("./runtime/plugin"));
   }
 });
 
diff --git a/dist/runtime/plugin.js b/dist/runtime/plugin.js
index 7c5a7f498bef6131e97504bd4fe22952b7d86a9c..221b6744484b5bf107a34ac3a4546f9d84808823 100644
--- a/dist/runtime/plugin.js
+++ b/dist/runtime/plugin.js
@@ -1,8 +1,6 @@
 import { convexVue } from "convex-vue";
 import { defineNuxtPlugin, useRuntimeConfig } from "#app";
 export default defineNuxtPlugin((nuxtApp) => {
-  const { url } = useRuntimeConfig().public.convex;
-  nuxtApp.vueApp.use(convexVue, {
-    url
-  });
+  const options = useRuntimeConfig().public['convex-vue'];
+  nuxtApp.vueApp.use(convexVue, options);
 });
