diff --git a/esm/src/Middleware.js b/esm/src/Middleware.js
index 796251bef63cef5148a2ce24c924edc10f6c1dc2..65c8fb4661771fcba50079fcb525a328bb5c7cc0 100644
--- a/esm/src/Middleware.js
+++ b/esm/src/Middleware.js
@@ -21,6 +21,9 @@ export function sessionMiddleware(options) {
         }
     }
     const middleware = createMiddleware(async (c, next) => {
+        if (typeof cookieOptions.expires === 'number')
+            cookieOptions.expires = new Date(Date.now() + (cookieOptions.expires * 1000))
+
         const session = new Session;
         let sid = '';
         let session_data;
