diff --git a/esm/src/Middleware.js b/esm/src/Middleware.js
index 52ecbc345c57a152f4ab5b131f70bc1a259e3375..81522df3c2b16b32e4e7ec5ba474778a5591334a 100644
--- a/esm/src/Middleware.js
+++ b/esm/src/Middleware.js
@@ -33,6 +33,12 @@ export function sessionMiddleware(options) {
         }
     }
     const middleware = createMiddleware(async (c, next) => {
+        if (typeof cookieOptions.expires === 'number')
+            globalThis.hsCExpires = cookieOptions.expires
+        
+        if (globalThis.hsCExpires)
+            cookieOptions.expires = new Date(Date.now() + (globalThis.hsCExpires * 1000))
+
         const session = new Session(expireAfterSeconds);
         let sid = '';
         let session_data;
